src/django/api/filters/submission.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework.filters import BaseFilterBackend
+
+from ..models.role import Roles
+
+
+class UserSubmissionFilterBackend(BaseFilterBackend):
+    def filter_queryset(self, request, queryset, view):
+        if request.user.role == Roles.CONTRIBUTOR:
```

Comment:
I just ran into the following problem myself, so I assume it applies here. (I'm thinking I should have gone with my original idea of making the `description` field the pk for `Role`.)

***
```python
>>> u.role
<Role: CONTRIBUTOR>
>>> u.role == Roles.CONTRIBUTOR
False
>>> u.role.pk == Roles.CONTRIBUTOR
True
```

Reply:
Could add something like this:
```diff
diff --git a/src/django/api/models/role.py b/src/django/api/models/role.py
index b532118..0ddf988 100644
--- a/src/django/api/models/role.py
+++ b/src/django/api/models/role.py
@@ -23,3 +23,6 @@ class Role(models.Model):
 
     def __str__(self):
         return self.description
+
+    def __eq__(self, other):
+        return self.pk.__eq__(other)
```

Reply:
https://github.com/azavea/iow-boundary-tool/pull/117